css/layout.css

```diff
@@ -2,33 +2,215 @@
   display: none !important;
 }
 
-.container {
-  max-width: 575px;
-  margin: 0 auto;
-  padding: 0 1.5rem;
+#container {
+  max-width: 860px;
+  margin-left: auto;
+  margin-right: auto;
 }
```

Comment:
It's almost never a good idea to use ids for styling. Ids are designed to be used once-per-id, which is great for hyperlinks (via links containing a hash) and efficient JavaScript targeting (`document.getElementById`), but poor for CSS. Whenever possible, use classes (or other HTML attributes) over ids for CSS targeting.